File: fastFramework/src/main/java/com/tesla/framework/common/util/LaunchTimer.java
package com.tesla.framework.common.util;

public class LaunchTimer {
    public static final String TAG = LaunchTimer.class.getSimpleName();

    private static long sTime;

    public static void startRecord() {
        sTime = System.currentTimeMillis();
    }

    public static void endRecord() {
        endRecord("");
    }

    public static void endRecord(String msg) {
        long cost = System.currentTimeMillis() - sTime;
        System.out.println(msg + "cost " + cost);
    }

}

File: fastFramework/src/main/java/com/tesla/framework/common/util/LaunchTimer.kt
package com.tesla.framework.common.util

import com.tesla.framework.component.logger.Logger

object LaunchTimer {
    val TAG = LaunchTimer::class.java.simpleName
    private var sTime: Long = 0
    fun startRecord() {
        sTime = System.currentTimeMillis()

    }

    @JvmOverloads
    fun endRecord(msg: String = "") {
        val cost = System.currentTimeMillis() - sTime
        Logger.d("$msg cost time: $cost ms")
    }
}